bitnami/keycloak/Chart.yaml

```
@@ -33,4 +33,4 @@ maintainers:
 name: keycloak
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/keycloak
-version: 19.3.3
+version: 19.3.4
```

As you are including a new parameter, please bump the chart to a new minor version

```suggestion
version: 19.4.0
```